### PR TITLE
Moves Scholarship Deadline Amount to Top of Campaign Info Card

### PR DESCRIPTION
**Pull request by Vmack**
### What's this PR do?

This pull request will move the scholarship amount to the top of campaign info card & make "Deadline" reflect the actual scholarship deadline. 

Small
Screen Shot 2020-02-18 at 1 37 04 PM

Medium
Screen Shot 2020-02-18 at 1 37 16 PM

Large
Screen Shot 2020-02-18 at 1 37 34 PM

### How should this be reviewed?

👀

### Any background context you want to provide?

This fix will allow more clarity for users and auto-update the deadline so that they no longer have to be updated manually.

### Relevant tickets

References [Pivotal #171053281](https://www.pivotaltracker.com/story/show/171053281).

### Checklist

- [ ] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added screenshots of front-end changes on small, medium, and large screens.
- [ ] Added appropriate feature/unit tests.

### CODE REVIEW

File: resources/assets/components/blocks/CampaignInfoBlock/CampaignInfoBlock.js [hunk 1]
```diff
@@ -27,28 +27,47 @@ const CAMPAIGN_INFO_QUERY = gql`
...
+                  <dd className="campaign-info__scholarship">
+                    {`$${scholarshipAmount.toLocaleString()}`}
+                  </dd>
 
-          if (!actionItem) {
-            actionItem = actions.find(action => action.reportback);
-          }
+                  <dt>Scholarship Deadline</dt>
+                  <dd>{getHumanFriendlyDate(scholarshipDeadline)}</dd>
+                  <hr className="clear-both pb-3 colors-gray-600" />
```
Comment (weerd) [inline resources/assets/components/blocks/CampaignInfoBlock/CampaignInfoBlock.js]: @Vmack `colors-gray-600` isn't a tailwind class. That's just the syntax for how it's used in the `tailwind()` helper function to grab that color when used in the CSS stylesheet. But when in the HTML you need to refer to the correct class!

Comment (Vmack) [reply resources/assets/components/blocks/CampaignInfoBlock/CampaignInfoBlock.js]: gotcha!

File: resources/assets/components/blocks/CampaignInfoBlock/CampaignInfoBlock.js [hunk 2]
```diff
@@ -27,28 +27,47 @@ const CAMPAIGN_INFO_QUERY = gql`
...
+                    Win A Scholarship
+                  </dt>
+                  <dd className="campaign-info__scholarship">
+                    {`$${scholarshipAmount.toLocaleString()}`}
+                  </dd>
 
-          if (!actionItem) {
-            actionItem = actions.find(action => action.reportback);
-          }
+                  <dt>Scholarship Deadline</dt>
```
Comment (katiecrane) [inline resources/assets/components/blocks/CampaignInfoBlock/CampaignInfoBlock.js]: Based on the designs I think we actually want this piece to say "Next Deadline" - does that seem right?

**Comment by katiecrane** [comment]
I think this is looking awesome! I checked out a [scholarship campaign](https://dosomething-phoenix-de-pr-1926.herokuapp.com/us/campaigns/test-teens-for-jeans) and a [non-scholarship campaign](https://dosomething-phoenix-de-pr-1926.herokuapp.com/us/campaigns/do-something-about-bugs) and they look great! Just had one comment about copy!

File: resources/assets/components/blocks/CampaignInfoBlock/CampaignInfoBlock.js (hunk 2 above)
Comment (Vmack) [reply resources/assets/components/blocks/CampaignInfoBlock/CampaignInfoBlock.js]: great catch, thanks! Should be good to go now!

**Pull request merged**